autobox.js

```diff
@@ -93,31 +94,18 @@ const CACHE_SIZE = 512
 
 function unboxWithCache (id) {
   var cache = HLRU(CACHE_SIZE)
-  // console.log('instantiating unbox.withCache', id)
 
-  // var state = {
-  //   total: 0,
-  //   cachedUnbox: 0,
-  //   unbox: 0
-  // }
   function cachedUnbox (msg, readKey, unboxers) {
-    // state.total++
-    // if (state.total % 1000 === 0) {
-    //   const percent = Math.floor(state.cachedUnbox / (state.cachedUnbox + state.unbox) * 1e3) / 1e2
-    //   console.log(`cache saves: ${percent}% (${state.cachedUnbox} / ${state.cachedUnbox + state.unbox})`)
-    // }
     if (!msg || !isString(msg.value.content)) return msg
 
     const cached = cache.get(msg.key)
-    // if (cached !== undefined) state.cachedUnbox++
-    // else state.unbox++
 
     if (cached === false && !readKey) return msg
     else if (cached) return cached
 
     const result = unbox(msg, readKey, unboxers)
     if (isString(result.value.content)) cache.set(msg.key, false)
-    else cache.set(msg.key, result)
+    else cache.set(msg.key, cloneDeep(result))
 
     return result
```

Comment:
So I'm still trying to understand what was going wrong with the streams.
Was it that later something is using the result and mutating the result (e.g. by running some `originalValue` or similar) which mutates the cached value?

That I can understand. What I can't understand is why restarting the server is what generates that state when you run a query which was previously running fine ... doesn't a server restart clear this LRU cache?
I see it passes the `ssb-tribes` test though ... so that's good.